### ISSUE TITLE
Cut nova requirement, causing composer issues

### ISSUE THREAD

**Issue by LorenzoSapora** [kirschbaum-development/nova-inline-select#1]
Composer spits out this when trying to `composer require [this-package]`

```php
  Problem 1
    - kirschbaum-development/nova-inline-select 1.0.1 requires laravel/nova ^2.0 -> no matching package found.
    - kirschbaum-development/nova-inline-select 1.0.0 requires laravel/nova ^2.0 -> no matching package found.
    - Installation request for kirschbaum-development/nova-inline-select ^1.0 -> satisfiable by kirschbaum-development/nova-inline-select[1.0.0, 1.0.1].

```

This is caused when following the manual install of nova (drop a folder in, and add manually to composer.json). However, if you use composer directly, this doesn't pop.

Considering that the main documented method is the manual method, I'd assume it's the most common installation method at the moment.

Going to push a PR for cutting the nova requirement from composer.json.

**Comment by brandonferens** [comment]
Thanks @LorenzoSapora. I'll check this out and look over your PR.

**Comment by brandonferens** [comment]
Merged #2

**Comment by eugenevdm** [comment]
Was this unmerged? I notice in composer.json still same requirement, making installation challenging. Please advise how to circumvent?
